lib/expand/join.go

```diff
@@ -830,7 +830,12 @@ func (p *Peer) tryConnect(operationID string) (ctx *operationContext, err error)
 			return nil, utils.Abort(err)
 		}
 		if trace.IsCompareFailed(err) {
-			p.printStep("Waiting for another operation to finish at %v", addr)
+			p.WithError(err).Info("Failed to create expand operation.")
```

Comment:
Nit: In a similar vein to what @knisbet suggested here before, maybe make this log message a bit less scary? Something like, "waiting on a precondition to create expand operation". Also, do we need full error trace here?